presto-main/src/main/java/com/facebook/presto/operator/TableWriterOperator.java

```diff
@@ -304,22 +320,31 @@ private Page createFragmentsPage()
 
         // output page will only be constructed once,
         // so a new PageBuilder is constructed (instead of using PageBuilder.reset)
-        PageBuilder page = new PageBuilder(fragments.size() + 1, ImmutableList.of(types.get(ROW_COUNT_CHANNEL), types.get(FRAGMENT_CHANNEL)));
+        PageBuilder page = new PageBuilder(fragments.size() + 2, ImmutableList.of(types.get(ROW_COUNT_CHANNEL), types.get(FRAGMENT_CHANNEL), types.get(CONTEXT_CHANNEL)));
         BlockBuilder rowsBuilder = page.getBlockBuilder(0);
         BlockBuilder fragmentBuilder = page.getBlockBuilder(1);
+        BlockBuilder operatorExecutionInfoBuilder = page.getBlockBuilder(2);
 
         // write row count
         page.declarePosition();
         BIGINT.writeLong(rowsBuilder, rowCount);
         fragmentBuilder.appendNull();
+        operatorExecutionInfoBuilder.appendNull();
 
         // write fragments
         for (Slice fragment : fragments) {
             page.declarePosition();
             rowsBuilder.appendNull();
             VARBINARY.writeSlice(fragmentBuilder, fragment);
+            operatorExecutionInfoBuilder.appendNull();
         }
 
+        // write operator execution info
+        page.declarePosition();
+        rowsBuilder.appendNull();
+        fragmentBuilder.appendNull();
+        VARBINARY.writeSlice(operatorExecutionInfoBuilder, operatorExecutionContextSlice);
```

Comment:
Do we want to have it as a `RunLengthEncodedBlock` as well? To be consistent with the layout of the pages that contain stats?